src/scripts/helpers/devLogger.ts

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
I am a little confused about the use case for these functions. I think we have Morgan for logging?, I am not familiar with it, but are we not able to use it with different log levels to keep production logs clean? (e.g. info, debug, warning, error etc.) 

For example, is there a risk that a developer might use devWarn or devError in a situation where we would actually want proper logging  and to be able to see and track the warning/error in production logs? Some of the devErrors above around invalid case types or error api response codes look like things we would want to log to production logs?